clang/lib/AST/OpenMPClause.cpp

```diff
@@ -305,6 +305,16 @@ OMPClause::child_range OMPIfClause::used_children() {
   return child_range(&Condition, &Condition + 1);
 }
 
+/*OMPClause::child_range OMPNowaitClause::used_children() {
+  return child_range(&Condition, &Condition + 1);
+}*/
+OMPClause::child_range OMPNowaitClause::used_children() {
+  if (Condition)
+    return child_range(&Condition, &Condition + 1);
+  Stmt *Null = nullptr;
+  return child_range(&Null, &Null);
```

Comment:
isn't this returning pointer to temporary? I'm unfamiliar with `child_range`, should it instead return `child_range(nullptr, nullptr)`?